flexmeasures_openweathermap/utils/locating.py

```diff
@@ -105,3 +106,15 @@ def find_weather_sensor_by_location_or_fail(
             % sensor_name
         )
     return weather_sensor
+
+
+def get_asset_id_location(asset_id: int) -> List[Tuple[float, float]]:
```

Comment:
Better name `get_location_by_asset_id`.

Also, it should return just the tuple, as we know there is only one.

Reply:
You can put in in a list in the CLI function before passing it on. Here, it makes this utility function quite confusing.

Clear naming and return types that fit expectations are very important for code quality.